pyaedt/edb_core/components.py

```diff
@@ -1199,13 +1217,13 @@ def set_component_rlc(self, componentname, res_value=None, ind_value=None, cap_v
             rlc.IsParallel = isparallel
             if res_value is not None:
                 rlc.REnabled = True
-                rlc.R = self._get_edb_value(res_value)
+                rlc.R = self._edb_value(res_value)
```

Comment:
I think here you should keep using the method `_get_edb_value` and not the property `_edb_value` used as a method.
I am assuming that this is causing the test to fail.
Same for the lines below.
```suggestion
                rlc.R = self._get_edb_value(res_value)
```